ROADMAP.md

```diff
@@ -1,7 +1,7 @@
 # KF Serving Roadmap
 ## 2019 - 2020
 
-### v0.4 Performance (ETA: Jan 31, 2019)
+### v0.4 Performance (ETA: Jan 31, 2020)
```

Comment:
@ellis-bigelow you wanna adjust the ETA?

Reply:
I think it will be better to adjust it since it confuses users like me.